.changeset/fair-rabbits-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/divider": patch
+---
+
+fix issues in RTL direction divider storybook
```

Comment:
@mrbadri we don't need to add a changeset for storybook changes

btw could you adjust the documentation example as well? https://nextui.org/docs/components/divider

Reply:
Hi [jrgarciadev](https://github.com/jrgarciadev), 
i resolve this issue